infrastructure/proto/log_replication_metadata.proto

```diff
@@ -129,7 +129,9 @@ message ReplicationEvent {
 
   enum ReplicationEventType {
```

Comment:
Observation: All the fields inside ReplicationEventType are force snapshot sync related.